src/wallet/rpc/spend.cpp

```diff
@@ -757,7 +757,10 @@ RPCHelpMan fundrawtransaction()
                 "Note that all inputs selected must be of standard form and P2SH scripts must be\n"
                 "in the wallet using importdescriptors (to calculate fees).\n"
                 "You can see whether this is the case by checking the \"solvable\" field in the listunspent output.\n"
-                "Only pay-to-pubkey, multisig, and P2SH versions thereof are currently supported for watch-only\n",
+                "Only pay-to-pubkey, multisig, and P2SH versions thereof are currently supported for watch-only\n"
```

Comment:
I see it needs a full stop after `watch-only`.

Reply:
```suggestion
                "Only pay-to-pubkey, multisig, and P2SH versions thereof are currently supported for watch-only.\n"
```